statics/js/components/capture-form.js

```diff
@@ -53,6 +53,13 @@ Vue.component('capture-form', {
             </div>\
             <div id="one" class="panel-collapse collapse">\
               <fieldset class="form-group">\
+                <div class="form-group" v-if="typeAllowed">\
```

Comment:
I would prefer to let the div displayed but marking the "select" input as disabled

Reply:
done